README.md

```diff
@@ -168,29 +170,62 @@ logger.with(a: 1) { logger.with(b: 2) { logger.info('Stuff') } }
 logger.with(a: 1, b: 2).info('Stuff')
 ```
 
-See the [class documentation](lib/roo_on_rails/context_logging.rb) for further details.
+See the [class documentation](lib/roo_on_rails/context_logging.rb) for further
+details.
 
 ### Google Oauth
 
 When `GOOGLE_AUTH_ENABLED` is set to true we'll:
 
-* Inject a `Omniauth` Rack middleware with a pre-configured strategy for Google Oauth2.
+* Inject a `Omniauth` Rack middleware with a pre-configured strategy for Google
+  Oauth2.
 * Onject custom Rack middleare to handle Oauth callback requests.
-* Generate the `config/initializers/google_oauth.rb` file that contains some examples of how to wire in your authentication logic.
+* Generate the `config/initializers/google_oauth.rb` file that contains some
+  examples of how to wire in your authentication logic.
 
 To use this functionality, you must:
 
-* Obtain the Oauth2 credentials from Google and configure them in `GOOGLE_AUTH_CLIENT_ID` and `GOOGLE_AUTH_CLIENT_SECRET`.
-* Provide in `GOOGLE_AUTH_ALLOWED_DOMAINS` a comma-separated list of domains, to whitelist the allowed email addresses.
-* Customize the code in the generated Rails initializer to hook into your application's authentication logic.
+* Obtain the Oauth2 credentials from Google and configure them in
+  `GOOGLE_AUTH_CLIENT_ID` and `GOOGLE_AUTH_CLIENT_SECRET`.
+* Provide in `GOOGLE_AUTH_ALLOWED_DOMAINS` a comma-separated list of domains, to
+  whitelist the allowed email addresses.
+* Customize the code in the generated Rails initializer to hook into your
+  application's authentication logic.
 * Update your Rails controllers to require authentication, when necessary.
 
 
+## Command features
+
+### Usage
+
+Run the following from your app's top-level directory:
+
+```
+bundle exec roo_on_rails
+```
+
+### Description
+
+Running the `roo_on_rails` command currently checks for:
+
+- the presence of `PLAYBOOK.md`
+- compliant Heroku app naming;
+- presence of the Heroku preboot flag;
+- correct Github master branch protection;
+- integration with the Heroku-Datadog metrics bridge (for CPU, memory, request
+  throughput data);
+- correct Sidekiq configuration.
+
+The command is designed to fix issues in many cases.
+
+
 ## Contributing
 
-Bug reports and pull requests are welcome on GitHub at https://github.com/deliveroo/roo_on_rails.
+Bug reports and pull requests are welcome on GitHub at
+`https://github.com/deliveroo/roo_on_rails`.
```

Comment:
I'd suggest we remove "bug reports" as issues are currently not enabled for this repository.